src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java

```diff
@@ -346,4 +339,17 @@ public static int checkDraftPipelineExistsViaApi(String draftId) throws IOExcept
     );
     return response.getResponseCode();
   }
+
+  public static void fillInPipelineName(String pipelineName) {
+    ElementHelper.clickOnElement(Helper.locateElementByTestId("pipeline-metadata"));
+    ElementHelper.clearElementValue(Helper.locateElementByCssSelector("#pipeline-name-input"));
+    ElementHelper.sendKeys(Helper.locateElementByCssSelector("#pipeline-name-input"), pipelineName);
+    ElementHelper.clickOnElement(
+      Helper.locateElementByTestId("pipeline-metadata-ok-btn"));
+  }
+  
+  public static void dismissTopBanner() {
+    ElementHelper.clickOnElement(Helper.locateElementByXPath(
+      "//div[@data-testid='valium-banner-hydrator']//button[@class='close ng-scope']"));
```

Comment:
maybe just add a `valium-banner-hydrator-close-button` datatestid

Reply:
I already added `banner-dismiss-button`, but that only works for failure banners. This was a tricky one to locate on the dom, I believe it has to do with the success banner disappearing after 3 seconds. Only worked with the xpath selector.